### DISCUSSION TITLE
Example Config

### DISCUSSION THREAD

**Discussion by SametBasturkk** [swar/Swar-Chia-Plot-Manager#218] [Q&A]
Can anyone share to me for 1 tb nvme example config.yaml?

Thanks

**Comment by Amneamnius** [comment]
That supports 3 parallel (4th with start_early). Based on the number of cores you can maybe do all 3 at the same time or stagger them.

I'd try to do 4 threads so about 16 threads total to allow all plots with a little bit of extra room if you are using the computer.
